jujugui/static/gui/src/app/components/accordion-section/accordion-section.js

```diff
@@ -1,7 +1,7 @@
 /* Copyright (C) 2017 Canonical Ltd. */
-
 'use strict';
 
+const SvgIcon = require('../svg-icon/svg-icon');
```

Comment:
the vast majority of these changes are strictly mechanical changes updating the copyright, and converting to use require() instead of YUI.